xontrib/autojump.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os
+import builtins
+import platform
+import subprocess
+from xonsh import dirstack
+from xonsh.events import events
+
+
+__all__ = ('_autojump', '_autojump_completer')
+
+_env = builtins.__xonsh_env__
```

Comment:
I don't think that a global reference to the Env object is a great idea because it could leak if this user decided to replace the env wholesale.  I think that it is probably best if this line is moved into each function body.